protocol/directory.go

```diff
@@ -39,18 +65,43 @@ func (d *ConiksDirectory) Update() {
 	}
 }
 
+// SetPolicies sets this ConiksDirectory's epoch deadline and VRF
+// private key.
 func (d *ConiksDirectory) SetPolicies(epDeadline merkletree.TimeStamp, vrfKey vrf.PrivateKey) {
 	d.policies = merkletree.NewPolicies(epDeadline, vrfKey)
 }
 
+// EpochDeadline returns this ConiksDirectory's current epoch deadline
+// as a timestamp.
 func (d *ConiksDirectory) EpochDeadline() merkletree.TimeStamp {
 	return d.pad.LatestSTR().Policies.EpochDeadline
 }
 
+// LatestSTR Returns this ConiksDirectory's latest STR.
```

Comment:
In the `EpochDeadline()` comment, it says "current" and here "latest".  We should probably just stick to one.